### PR TITLE
Add flag option for count setting

### PR DESCRIPTION
**Pull request by nkprince007**
this is a step in solving #95

### CODE REVIEW

File: uiplib/UIP.py [hunk 1]
```diff
@@ -14,9 +14,17 @@ def main():
                              " and downloads new ones. "
                              "When combined with --offline,"
                              " deletes the wallpapers and exits.")
+    parser.add_argument("--count",
```
Comment (abhsag24) [inline uiplib/UIP.py]: also @teja-315 is working on something similar right?

**Comment by abhsag24** [comment]
@nkprince007  i think you should wait for @teja-315 's implementation and then just add a flag for her setting?

File: uiplib/UIP.py (hunk 1 above)
Comment (nkprince007) [inline uiplib/UIP.py]: But do lemme know what you want the argument to be.. :)

**Comment by abhsag24** [comment]
@nkprince007 merged @teja-315 's commit now you have conflicts :D

File: uiplib/UIP.py (hunk 1 above)
Comment (nkprince007) [inline uiplib/UIP.py]: Ok, will resolve and push..

File: uiplib/scrape.py [hunk 2]
```diff
@@ -28,14 +28,17 @@ def dlProgress(count, blockSize, totalSize):
...
+def get_images(url, directory, count):
     '''
     scrape images from /r/wallpapers
     '''
     image_links = []
     p_url = url
-    while len(image_links) < NUMBER_OF_IMAGES_TO_PARSE:
+
+    # this specifies the number of images to be parsed
+    no_of_images = NUMBER_OF_IMAGES_TO_PARSE if not count else int(count)
```
Comment (abhsag24) [inline uiplib/scrape.py]: hey can u make this change in setup.py to add NUMBER_OF_IMAGES_TO_PARSE in the `settings.json` so we can remove this from `settings.py` and have a default value for that if no one specified this flag?

Comment (nkprince007) [inline uiplib/scrape.py]: Sure, bro :)

Comment (abhsag24) [inline uiplib/scrape.py]: thats the magic of the new `settings.py` :P default values :D

**Pull request merged**